components/automate-ui/src/app/entities/projects/project.effects.ts

```diff
@@ -196,7 +201,19 @@ export class ProjectEffects {
           observableOf(new GetApplyRulesStatusFailure(error))))));
 
   @Effect()
-  getLatestApplyRulesStatus$ = observableInterval(1000 * POLLING_INTERVAL_IN_SECONDS)
-    .pipe(map(() => new GetApplyRulesStatus()));
+  getLatestApplyRulesStatus$ = observableInterval(1000 * POLLING_INTERVAL_IN_SECONDS).pipe(
+    withLatestFrom(this.store.select(iamMajorVersion).pipe(filter(identity))),
+    withLatestFrom(this.store.select(iamMinorVersion).pipe(filter(identity))),
+    withLatestFrom(this.store.select(allPerms)),
+    filter(([[[_, major], minor], perms]) => {
+      const permObject = perms[ProjectConstants.APPLY_RULES_ENDPOINT];
+      const allowed = permObject && permObject.get;
```

Comment:
I agree, the last is a bit much for all-in-one-line.
As to the former, the reason I did not go with that (or with the equivalent `const allowed = perms[ProjectConstants.APPLY_RULES_ENDPOINT].get;`) is that it does not allow for the case of the permObject being undefined.
While rather unlikely, it could happen if for some odd reason the dispatch-on-startup that populates this particular perm got delayed. That is why I have the extra guard in L210.